staging/src/k8s.io/kubelet/pkg/apis/stats/v1alpha1/types.go

```diff
@@ -65,9 +65,9 @@ type NodeStats struct {
 type RlimitStats struct {
 	Time metav1.Time `json:"time"`
 
-	// The max PID of OS.
+	// The max number of extant process (threads, precisely on Linux) of OS.
```

Comment:
`extant`? not sure what the right word here should be.

Reply:
It's a _perfectly cromulent_ word but not very widely used.
Maybe:
```suggestion
 	// The operating system ceiling on the number of process IDs that can be assigned.
 	// On Linux, tasks (either processes or threads) consume 1 PID each.
```

Reply:
TIL : )

I copied them from https://man7.org/linux/man-pages/man2/getrlimit.2.html

```
       RLIMIT_NPROC
              This is a limit on the number of extant process (or, more
              precisely on Linux, threads) for the real user ID of the
              calling process.  So long as the current number of
              processes belonging to this process's real user ID is
              greater than or equal to this limit, fork(2) fails with
              the error EAGAIN.
```

I really want to make it crystal clear here as opposed to previous comment that this value is for the count of running tasks the OS allows at any point instead of just a value range for the PID, although the latter also implies the same limitation :-

Reply:
Maybe combine both? Keep extant, maybe with the link, and add the extended comment version suggested by @sftim.